packages/@okta/vuepress-site/docs/reference/webfinger/index.md

```diff
@@ -2,7 +2,7 @@
 title: WebFinger
 meta:
   - name: description
-    content: The Webfinger protocol helps to determine the identity provider where a given username should be routed. This guide explains the process of finding a user's identity provider.
+    content: The Webfinger protocol helps to determine the identity provider where a given username should be routed. This guide explains the process of finding the identity provider for a user.
```

Comment:
style is to capitalize Identity Provider:

The Webfinger protocol helps to determine the Identity Provider where a given username should be routed. This guide explains the process of finding the Identity Provider for a user.

Reply:
Fixed